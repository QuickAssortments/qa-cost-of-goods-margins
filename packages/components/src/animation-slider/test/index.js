/** @format */
/**
 * External dependencies
 */
import renderer from 'react-test-renderer';

/**
 * Internal dependencies
 */
import AnimationSlider from '../';

describe( 'AnimationSlider', () => {
	test( 'it renders correctly', () => {
		const tree = renderer
			.create(
				<AnimationSlider animationKey={ 0 } animate={ 'left' }>
					{ () => <div /> }
				</AnimationSlider>
			)
			.toJSON();
		expect( tree ).toMatchSnapshot();
	} );
} );
