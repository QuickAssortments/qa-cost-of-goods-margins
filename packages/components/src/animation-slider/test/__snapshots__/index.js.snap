// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimationSlider it renders correctly 1`] = `
<div
  className="woocommerce-slide-animation animate-left"
>
  <div>
    <div />
  </div>
</div>
`;
